package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-react": "^7.0.0",
     "eslint-plugin-standard": "^3.0.1",
     "md5-file": "^3.2.3",
-    "mocha": "^5.1.1",
+    "mocha": "^6.2.3",
```

Comment:
Does this mean that the `minimist` vulnerability warning was resolved upstream? i.e. we don't need to force a resolution any more?

Reply:
Yes, exactly. This new mocha version was released recently contains the correction of the vulnerability problem: https://github.com/mochajs/mocha/releases/tag/v6.2.3. 
Furthermore, the pipeline is green. If there would have been some `audit` problem, then it would be red.

Reply:
An alternative mocha version is also available with the correction: https://github.com/mochajs/mocha/releases/tag/v7.1.2
It contains the latest `mkdirp` version: 0.5.5 This `mkdirp` package uses `minimist` package. `mkdirp` v0.5.4  is the first version where the corrected `minimist` in use and this version is used in mocha v6.2.3

Reply:
OK well we could just update to 7.1.2 then if the `6`->`7` doesn't cause any failures.